pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -578,6 +578,18 @@ public SimpleHttpResponse sendDeleteRequest(URI uri, String authToken)
     return sendRequest(requestBuilder.build());
   }
 
+  public SimpleHttpResponse postJsonRequest(URI uri, String jsonRequestBody, String authToken)
```

Comment:
Annotate `jsonRequestBody` as nullable.

Side question: Could `authToken` be null? Will it work properly when it is null?